### ISSUE TITLE
The toolbar doesn't stay within the viewport

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5-editor-inline#10]
![apr-10-2017 19-29-58](https://cloud.githubusercontent.com/assets/156149/24874238/47467b3a-1e24-11e7-8b80-21fa86e4e091.gif)

**Comment by oleq** [comment]
Yeah. This feature is missing. OTOH, it's not very critical because it's not a very common case when the editor spans entire screen height.

**Comment by Reinmar** [comment]
> OTOH, it's not very critical because it's not a very common case when the editor spans entire screen height.

Isn't it the most popular case? ;| Just like you can see in the demo. This is a very typical implementation.

**Comment by fredck** [comment]
This is very critical, indeed.

**Comment by oskarwrobel** [comment]
Related: https://github.com/ckeditor/ckeditor5-ui/issues/181

**Comment by oleq** [comment]
It turned out that it was super easy to implement. Please take a look on https://github.com/ckeditor/ckeditor5-editor-inline/pull/12.
